static/app/views/starfish/queries/useSpanTransactionMetrics.tsx

```diff
@@ -26,35 +25,33 @@ export type SpanTransactionMetrics = {
 };
 
 export const useSpanTransactionMetrics = (
-  span: Pick<IndexedSpan, 'group'>,
+  group: string,
```

Comment:
m: should we rename to either `group` or `groupId` across the board?

Reply:
It should be `group` everywhere, I just chose not to do it in this particular PR